app/views/mentors/_form.html.erb

```diff
@@ -18,17 +18,17 @@
   <div class="card p-5" style="width: 48rem;">
     <div class="form-group">
       <%= form.label :first_name, class: "input-group mb-3" %>
-      <%= form.text_field :first_name, class: "form-control" %>
+      <%= form.text_field :first_name, class: "form-control", value: current_user.first_name %>
     </div>
 
     <div class="form-group">
       <%= form.label :last_name, class: "input-group mb-3" %>
-      <%= form.text_field :last_name, class: "form-control" %>
+      <%= form.text_field :last_name, class: "form-control", value: current_user.last_name %>
     </div>
```

Comment:
I'm about to put in a PR to set the user's name based on their email. I don't think this should be part of the form, although we could add a "preferred name" field if we want to be super sensitive.

Reply:
Hah I think I just paraphrased what you said in the other pr.. agree 💯. Let's follow up later for this